src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.model;
+
+public final class FruitTransaction {
```

Comment:
Consider making the `FruitTransaction` class immutable by removing the setters and making the fields `final`. This can help ensure that instances of this class remain consistent and unchanged once created.